lib/view/pages/home_page.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:superformula_test/core/resources/extensions.dart';
+import 'package:superformula_test/view/widgets/extended_menu_fab.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  double _imageScale = 0;
+  double _textXOffset = -3;
+
+  @override
+  void initState() {
+    super.initState();
+    setAnimationTimer();
+  }
+
+  void setAnimationTimer() {
+    Timer(const Duration(milliseconds: 100), () {
+      setState(() {
```

Comment:
**note**: this `setState` call rebuild the entire subtree on each tick. It also rebuilds widgets that should not be rebuilt (such as `Scaffold`, `Center` and `Column`) because they do not depend on the animation. 🙂 

If you used an `AnimationController` you could improve the code and reduce rebuilds to only those widgets that actually need it.

Reply:
This will only happen one time, but still not a good approach. I will keep attention to that! Thanks :D